litellm/integrations/literal_ai.py

```diff
@@ -102,7 +99,6 @@ def _send_batch(self):
                 },
                 headers=self.headers,
             )
-            response.raise_for_status()
```

Comment:
any reason to remove `raise_for_status` ?

Reply:
it seems the `HttpHandler` class already does it out of the box